_posts/2003-01-29-008.md

```diff
@@ -8,7 +8,7 @@ tags:
 - xanga
 comments: true
 image: "/assets/images/xanga/2004-04-16.jpg"
-slug: 008
+slug: '008'
```

Comment:
These ones are epic! Without quotes these slugs were treated by YAML as numbers and not strings!!! Which lead to weird error messages and without debugging it with `pry` I would not be able to figure out what the heck was going on...